react/utils/GetString.js

```diff
@@ -1,7 +1,11 @@
 export function getPickupSlaString(slas) {
   return slas.reduce(
     (accumulatedString, currentPickupPoint) =>
-      currentPickupPoint.id ? accumulatedString + currentPickupPoint.id : '',
+      currentPickupPoint.id
+        ? accumulatedString +
+          currentPickupPoint.id +
+          currentPickupPoint.pickupDistance
```

Comment:
<img width="517" alt="Captura de Tela 2022-07-01 às 10 16 56" src="https://user-images.githubusercontent.com/5691711/176904544-2d6d5e2a-f2aa-4b32-ae00-fea57c948bed.png">
this function is only being called by a single place